### PR TITLE
Assignment3 & Assignment4

### PR DESCRIPTION
**Pull request by JustAntoRS**
All the queries have been validated in the dbpedia endpoint and .csv files are the same ones downloaded from the endpoint.

### CODE REVIEW

**Comment by JustAntoRS** [comment]
Added Assignment 4 to the PR

File: Assignment4/JustAntoRS-z170035/Task06.ipynb [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Add the University to the graph\n",
+        "g.add((ns.UPM, RDF.type, ns.University))\n",
+        "# John Smith works in UPM\n",
+        "g.add((ns.JohnSmith, vcard.Work, ns.UPM))"
```
Comment (dachafra) [inline Assignment4/JustAntoRS-z170035/Task06.ipynb:188]: this property does not exist in vcard vocabulary, and a colleague from you made the same mistake ;-)

**Review by dachafra — CHANGES_REQUESTED** [review]
assign4: task07 correct, task06 needs a minor change

**Comment by dachafra** [comment]
@ocorcho your time for assign3

**Comment by ocorcho** [comment]
Assignment 3 is ok, so as soon as the minor edit in the task is done, this can be accepted.

File: Assignment4/JustAntoRS-z170035/Task06.ipynb (hunk 1 above)
Comment (JustAntoRS) [reply Assignment4/JustAntoRS-z170035/Task06.ipynb:188]: Yes, I tried searching for a property for working in the web "http://www.w3.org/2001/vcard-rdf/3.0#" and I failed at it :sweat_smile: so I took a look at what property other students have used here.

Thanks for spotting this out! :)

**Comment by JustAntoRS** [comment]
@dachafra I see that the property `vcard.Work` does not exist but when looking into: http://www.w3.org/2001/vcard-rdf/3.0# I see that there is this property:
```
<rdfs:Class rdf:about="TELTYPES"/>
<vcard:TELTYPES rdf:about="home"/>
<vcard:TELTYPES rdf:about="msg"/>
<vcard:TELTYPES rdf:about="work"/>
<vcard:TELTYPES rdf:about="fax"/>
<vcard:TELTYPES rdf:about="cell"/>
<vcard:TELTYPES rdf:about="video"/>
<vcard:TELTYPES rdf:about="pager"/>
<vcard:TELTYPES rdf:about="bbs"/>
<vcard:TELTYPES rdf:about="modem"/>
<vcard:TELTYPES rdf:about="car"/>
<vcard:TELTYPES rdf:about="isdn"/>
<vcard:TELTYPES rdf:about="pcs"/>
```

Where I see there is the `work` property, how am I supossed to use this property?

Note: Sorry for adding a dummy commit about the HandsOn Assignment 1, I did git push before checking in which branch I was working on :sweat_smile:

**Comment by dachafra** [comment]
Hi @JustAntoRS,
The only "Work" I see in vcard is the class one: https://www.w3.org/2006/vcard/ns#Work and you cannot use something that is a class (usually you can be known that is a class bc it starts with a capital letter) in the predicate please, so please use your own namespace (ns for example) to define this property

**Comment by dachafra** [comment]
The PR is going to be merged although there is that minor error.

**Review by dachafra — APPROVED** [review]

**Comment by JustAntoRS** [comment]
Thanks! I am going to make a new PR to merge the HandsOn assignment and solve the minor issue

**Pull request merged**